### DISCUSSION TITLE
NDB Cluster Driver - Unknown system variable 'query_cache_size'

### DISCUSSION THREAD

**Discussion by gary-ionline** [dbeaver/dbeaver#18463] [Q&A]
I am trying to connect to a MySQL 8 NDB cluster using the NDB Cluster driver.

When I setup the connection and attempt to connect, I get the error:
Unknown system variable 'query_cache_size'

Can anyone help with this? I've been forced to go back to MySQLWorkb**ch because of this error

I've searched and the suggestions have been to update the MySQL driver as query_cache_size has been removed from MySQL 8. Olders suggestions found are to upgrade to JDBC driver 5.1.44. But I'm already on Driver MySQL 8.0.31 so this should not be a problem.

**Env**
- MacOS Monterey v12.2.1 Apple M1 2020
- DBBeaver Version 22.3.0.202212041619 
- Driver ID: mysql_ndb
- Driver Class name: com.mysql.jdbc.Driver
- Driver File: ../Library/DBeaverData/drivers/maven/maven-central/mysql/mysql-connector-java-8.0.31.jar

Has anyone encountered this and how did you manage to fix it? Is it to do with my local drivers or a change required on the target DB server?

**Comment by LonwoLonwo** [comment]
Could you provide the log files? You can find instructions here: https://dbeaver.com/docs/wiki/Log-files/. Thank you.
